docs/quickstart.html

```diff
@@ -58,8 +58,12 @@ <h2 class="title-2">Installing</h2>
     <code class="code">~/.please</code> but you can put it where you want.
   </p>
 
+  <p>See <a href="faq.html#verify">here</a> for information if you'd like to verify
+    signatures of the download before running.</a>
```

Comment:
`</a>` => `</p>`

Reply:
I do know the difference between `a` and `p`, honestly... :-P